input/fsh/PatientIdentifierPatternConformanceSE.fsh

```diff
@@ -0,0 +1,29 @@
+Profile: PatientIdentifierPatternConformanceSE
+Parent: PatientSEVendorLite
+Id: PatientIdentifierPatternConformanceSE

```

Comment:
If Id and profile name are the same, as in this case, Id can be excluded and it will be automatically taken from profile name